infrastructure/zk/src/test/test.ts

```diff
@@ -4,6 +4,11 @@ import * as utils from '../utils';
 import * as integration from './integration';
 export { integration };
 
+const CODE_COVERAGE_ENABLED = process.env.CODE_COVERAGE;
+const CODE_COVERAGE_FLAGS = `CARGO_INCREMENTAL=0 RUSTFLAGS='-C instrument-coverage' LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw'`;
+
+const codeCoverageFlags = () => (CODE_COVERAGE_ENABLED ? CODE_COVERAGE_FLAGS : '');
```

Comment:
not extremely important, but why is this a function if it only deals with constants?
wouldn't this do?
```ts
const CODE_COVERAGE_FLAGS = process.env.CODE_COVERAGE
    ? `CARGO_INCREMENTAL=0 RUSTFLAGS='-C instrument-coverage' LLVM_PROFILE_FILE='cargo-test-%p-%m.profraw'`
    : '';
```

Reply:
https://github.com/rsksmart/rif-rollup/pull/94/files#r1324815783 ... unless we do `JSON.parse(process.env.CODE_COVERAGE)` ;)

Reply:
I'd prefer to explicitly check that the value specified is either `true` or `1` and applying the flags only in these cases.

Reply:
Done